easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -318,8 +318,29 @@ def configure_step(self, srcdir=None, builddir=None):
 
         (out, _) = run_cmd(command, log_all=True, simple=False)
 
-        return out
-
+        # sanitycheck for the configuration step
+        self.log.info("Checking python paths")
+        with open('CMakeCache.txt', 'r') as file:
```

Comment:
This should handle the case that this file may not exist. E.g. when `configure_cmd != DEFAULT_CONFIGURE_CMD` anything can happen.